packages/load-script/src/dom-operations.js

```diff
@@ -3,14 +3,20 @@ import { handleRequestError, handleRequestSuccess } from './callback-handler';
 
 const debug = debugFactory( 'lib/load-script/dom-operations' );
 
-export function createScriptElement( url ) {
+export function createScriptElement( url, args ) {
 	debug( `Creating script element for "${ url }"` );
 	const script = document.createElement( 'script' );
+
+	if ( args ) {
+		Object.entries( args ).forEach( ( [ key, value ] ) => script.setAttribute( key, value ) );
+	}
```

Comment:
Should this happen after the 5 `script.${attribute}` assignment lines below? Would be helpful if we want to override one of them specifically. 

If you agree with me and decide to go that way, this could also use a couple of tests to ensure that the precedence works well, and for example, `async` is true by default, but we can set it to `false` through `args`.

Reply:
I was thinking that we'd want to prevent overriding those defaults, but it's probably not necessary. I'll make the changes.

Reply:
OK, funny thing. I've done the change, but the test fails - async stays true, even though I used `script.removeAttribute()`. This works in the browser console, but not here and I'm not sure why.